ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -64,8 +64,7 @@
   @SuppressWarnings("FieldMayBeFinal")
   @Option(
       names = {"--downstream-http-request-timeout"},
-      description =
-          "Timeout in seconds to wait for downstream request to timeout (default: ${DEFAULT-VALUE})",
+      description = "Timeout in seconds to wait for downstream request (default: ${DEFAULT-VALUE})",
```

Comment:
nit: wait for the request - or response? (and should this be in the option name too?)